### PR TITLE
Add 11ty faq

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the goal of this pull request and the changes that were made.

Create FAQ page for Quire developments and transition to 11ty.

### CODE REVIEW

File: content/about/faq.md [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
+Eleventy (11ty) is a simple yet powerful [static site generator](https://quire.getty.edu/about/how-it-works/) based on JavaScript. It was created as an alternative to [Jekyll](https://jekyllrb.com/) and will replace our former static site generator, [Hugo](https://gohugo.io/). 11ty transforms a directory of various templates into HTML and is popular for its ability to mix multiple templating languages (including: HTML, Markdown, JavaScript, Liquid, Nunjucks, Handlebars, Mustache, EJS, Haml, Pug, and Custom). It is available as open source under the MIT License. Read more on the [11ty website](https://www.11ty.dev/).
```
Comment (geealbers) [inline content/about/faq.md:10]: I'd eliminate the mention of Jekyll, I don't think it adds anything.

I'd also remove the parenthetical list of template types. While yes it supports all those, Quire will only use Liquid and JavaScript

File: content/about/faq.md [hunk 2]
```diff
@@ -1,23 +1,54 @@
...
+### What are the benefits of 11ty?
 
-In Latin:
-> *Quire is also a form of the Latin verb “queo”, or “I am able” or “I can”. Specifically, it’s the 2nd principal part and so means “to be able”. It’s also intransitive and expects an action. To be able to write, “quire scribere”. To be able to publish, “quire prodere”. To be able to edit, “quire corrigere”. Quire literally “can” do anything.*
+11ty has a significantly faster processing time than other static site generators. With the move to 11ty, we have reworked the overall file structure of Quire projects to simplify navigation and make essential project files more accessible. On the development side, one of 11ty's great strengths is its ability to mix templating languages. For example, in Quire projects, Liquid will be used to generate simple markup while JavaScript will be used where data manipulation is required (more on those two langauges below.) Other templating languages may also be used. Above all, the move to 11ty will enable the expansion of advanced features, a more robust set of CLI options, and improve developer support and contribution opportunities.
```
Comment (geealbers) [inline content/about/faq.md:14]: Hugo is actually also very fast, fyi. It's all the other stuff that slows things down. That will likely be the same with us and 11ty.

File: content/about/faq.md [hunk 3]
```diff
@@ -1,23 +1,54 @@
...
 
-**Who makes up the Quire community?**
+### What is JavaScript?
 
-**Why Open Source?**
+Javascript is a popular text-based programming language that turns static web pages into dynamic, interactive sites, going a step beyond the capabilities of HTML and CSS. JavaScript can be found everywhere on the web, from refreshing a Twitter feed to your Amazon search bar or embedding Youtube videos. It is also widely used in the art, cultural heritage, and academic sectors. Learn more about JavaScript on the [Mozilla developer website](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript).
```
Comment (geealbers) [inline content/about/faq.md:18]: JavaScript, with capital S in start of paragraph.

File: content/about/faq.md [hunk 4]
```diff
@@ -1,23 +1,54 @@
...
 
-**How can I participate?**
+### What is Liquid?
+
+Liquid is an open-source templating language written in [Ruby](https://www.ruby-lang.org/en/). It was created by the e-commerce company [Shopify](https://www.shopify.com/) in 2006. It is the backbone of all Shopify themes and is used to load dynamic content on website storefronts. Read more about Liquid on the [Shopify.dev website](https://shopify.dev/api/liquid).
+
+### What is Vite
+
+[Vite (Vite.js)](https://vitejs.dev/) is a lightning fast tool for frontend development. It is replacing [Webpack](https://webpack.js.org/) and will be used to compile styles and scripts as well as bundle files.
```
Comment (geealbers) [inline content/about/faq.md:26]: "lightning fast tool for frontend development" sounds copy-pasted straight from their website. I'm not 100% clear on it myself though, maybe worth asking the team?

File: content/about/faq.md [hunk 5]
```diff
@@ -1,23 +1,54 @@
...
 
-**Why Open Source?**
+Javascript is a popular text-based programming language that turns static web pages into dynamic, interactive sites, going a step beyond the capabilities of HTML and CSS. JavaScript can be found everywhere on the web, from refreshing a Twitter feed to your Amazon search bar or embedding Youtube videos. It is also widely used in the art, cultural heritage, and academic sectors. Learn more about JavaScript on the [Mozilla developer website](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript).
 
-**How can I participate?**
+### What is Liquid?
+
+Liquid is an open-source templating language written in [Ruby](https://www.ruby-lang.org/en/). It was created by the e-commerce company [Shopify](https://www.shopify.com/) in 2006. It is the backbone of all Shopify themes and is used to load dynamic content on website storefronts. Read more about Liquid on the [Shopify.dev website](https://shopify.dev/api/liquid).
+
+### What is Vite
```
Comment (geealbers) [inline content/about/faq.md:24]: Needs a question mark

File: content/about/faq.md [hunk 6]
```diff
@@ -1,23 +1,54 @@
...
+
+[Vite (Vite.js)](https://vitejs.dev/) is a lightning fast tool for frontend development. It is replacing [Webpack](https://webpack.js.org/) and will be used to compile styles and scripts as well as bundle files.  
+
+### What is our timeline for transitioning to 11ty?
+
+We expect the bulk of development work to be done by July 2022 with Quire's official open-source launch scheduled for the Fall. We look to the Quire community for feedback and to help us spot bugs that the transition may have generated. The move will be a gradual work in progress, and we appreciate our users' patience.
+
+### How does this affect older or in process Quire projects?
```
Comment (geealbers) [inline content/about/faq.md:32]: in-process should be hyphenated

File: content/about/faq.md [hunk 7]
```diff
@@ -1,23 +1,54 @@
...
+
+We are looking for contributions to help support the creation of developer-centric documentation. If you are interested in contributing please see our [Contributor Guidelines](https://github.com/thegetty/quire-docs/blob/main/CONTRIBUTING.md).
+
+### Can I explore 11ty for myself?
+
+We will have an 11ty branch of Quire available for people to play around with and offer feedback in the coming weeks.
+
+### What are some other Quire developments underway?
+
+In addition to our work on 11ty, we are also working on switching our image viewing from Leaflet to [CanvasPanel](https://canvas-panel.digirati.com/#/about). We are also greatly improving our IIIF capabilities, including adding the ability to annotate images and layer images. We are also working to replace Prince XML with the open-source PDF generator, [Paged.js](https://pagedjs.org/). All this development work is still underway. We thank you for your patience.
```
Comment (geealbers) [inline content/about/faq.md:50]: "including adding the ability to annotate images and layer images" isn't quite right as I think anotations would have to be made outside Quire. Maybe just leave as "including adding the ability to layer images" for now

File: content/about/faq.md [hunk 8]
```diff
@@ -1,23 +1,54 @@
 ---
-title: FAQ
+title: FAQs
```
Comment (geealbers) [inline content/about/faq.md:2]: Maybe need a little preamble/abstract at the beginning to clarify that these are specifically FAQs about our move to 11ty?

**Review by geealbers — APPROVED** [review]
Nice job. It feels pretty meaty.

What do you think about exposing it more on the site? Like putting it in the side navigation and/or even adding a banner to the site home page or maybe even every page to alert folks to the upcoming changes? I'm just thinking that the notification and FAQs will be just as relevant to someone who comes to the site/Quire two weeks from now as it is to those who get and open the newsletter.

**Comment by Erin-Cecele** [comment]
@geealbers I made the requested adjustments and also surfaced the FAQ page so it's no longer hidden. A banner on the homepage is also probably a good idea too. Our conversation with Paige today definitely illustrated that.

File: content/about/faq.md [hunk 9]
```diff
@@ -1,23 +1,77 @@
...
+
+### What is Liquid?
+
+Liquid is an open-source templating language written in [Ruby](https://www.ruby-lang.org/en/). It was created by the e-commerce company [Shopify](https://www.shopify.com/) in 2006. It is utilized in all Shopify themes to load dynamic content on website storefronts. Learn more about Liquid on the [Shopify.dev website](https://shopify.dev/api/liquid).
+
+### What is Vite?
+
+Vite is a JavaScript bundler and development web server. It will replace Webpack as a tool to bundle scripts and styles for the web. Learn more on the [Vite website](https://vitejs.dev/).
+
+### What is CanvasPanel?
```
Comment (geealbers) [inline content/about/faq.md:48]: The link you found about Canvas Panel is outdated. Let's use this one: https://iiif-canvas-panel.netlify.app/. Also note that they seem to have added a space to the name so it's "Canvas Panel" and not "CanvasPanel". Sorry I didn't catch it before.

**Review by geealbers — APPROVED** [review]
@Erin-Cecele it looks good. I had one note about Canvas Panel. Feel free to merge it after that's addressed. Thanks!

**Pull request merged**